docs/bootloaders.md

```diff
@@ -0,0 +1,41 @@
+---
+nav_order: 11
+---
+
+# Bootloaders
```

Comment:
I think markdown linters won't like two toplevel `#`; simplest is probably to make everything else `##`